.github/dependabot.yml

```diff
@@ -6,4 +6,4 @@ updates:
     schedule:
       interval: weekly
     commit-message:
-      prefix: DEV
+      prefix: gh-725
```

Comment:
How about `dependabot: ` for the title? [We can track dependabot already.](https://github.com/glass-dev/glass/issues?q=is%3Apr+author%3Aapp%2Fdependabot)

Reply:
Okay sure. Wasn't sure what you wanted.

Reply:
Yep, I remember what the codebase looked like when we started on it!